### PR TITLE
clean-up

### PR DESCRIPTION
**Pull request by retokromer**
- use modern shebang
- use `$( )` instead of `` ` ` ``
- alignment & spacing

### CODE REVIEW

File: admin [hunk 1]
```diff
@@ -1,64 +1,62 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (reneewiles2323) [inline admin]: good one

**Comment by retokromer** [comment]
ping

**Comment by Jojo-Schmitz** [comment]
6 space indent is MuseScore coding style

**Comment by retokromer** [comment]
done

File: admin [hunk 2]
```diff
@@ -30,28 +30,27 @@ case $1 in
             if [ ! -d .git ]; then
                   echo "Error: must run this script from the root of a git repository"
                   exit 1
-                  fi
+            fi
```
Comment (Jojo-Schmitz) [inline admin]: This 'banner style' indent is MuseScore coding style

File: admin [hunk 3]
```diff
@@ -60,5 +59,4 @@ case $1 in
             echo "gitremove a  -- permanently remove file/folder a"
             echo "gitupdate    -- pull from master"
             ;;
-      esac
-
+esac
```
Comment (Jojo-Schmitz) [inline admin]: This 'banner style' indent is MuseScore coding style

**Comment by retokromer** [comment]
OK

**Comment by Jojo-Schmitz** [comment]
No need to close the PR, the rest of seems good and valid

**Comment by lasconic** [comment]
Hi,
Thank you for your interest in contributing to MuseScore. I checked with @wschweer, the creator of this script, since many variables are linked to his machine and he doesn't use this script anymore. So I will just delete it in the next commit. No need to fix the syntax.

**Pull request closed without merging**